src/components/projekti/common/KansalaisenAineistoNakyma.tsx

```diff
@@ -17,6 +17,49 @@ import { ButtonFlatWithIcon } from "@components/button/ButtonFlat";
 import { H3, H4, H5 } from "@components/Headings";
 import { AineistoLinkkiLista } from "../kansalaisnakyma/AineistoLinkkiLista";
 import { isDateTimeInThePast } from "backend/src/util/dateUtil";
+import Notification, { NotificationType } from "../../notification/Notification";
+
+type AccordioSummaryContentProps = {
+  titleText: string;
+  paakategoria?: boolean;
+  tooltipText?: string;
+};
+
+const AccordioSummaryContent = ({ titleText, paakategoria, tooltipText }: AccordioSummaryContentProps) => {
+  const [showTooltip, setShowTooltip] = useState(false);
+  const Heading = paakategoria ? H4 : H5;
+  return (
+    <div className="w-full">
+      <div className="flex flex-row items-center justify-between max-w-sm">
+        <Heading variant="h5" sx={{ margin: 0, alignSelf: "baseline" }}>
+          {titleText}
+        </Heading>
+        {tooltipText && (
+            <FontAwesomeIcon
+              color="rgb(0, 100, 175)"
+              size={"lg"}
+              icon="info-circle"
+              type={NotificationType.INFO_GRAY}
+              cursor="pointer"
+              onMouseEnter={() => setShowTooltip(true)}
+              onMouseLeave={() => setShowTooltip(false)}
+              onClick={(event) => {event.stopPropagation(), setShowTooltip(true)}}
```

Comment:
Varmaan näin event.stopPropagation(); setShowTooltip(true);

Reply:
Korjattu

Reply:
Tuolla on kyllä edelleen pilkku kuten myös alempana Notification komponentin onClose käsittelijässä. Sonarlint noista mussuttaa. Meidän linter on konfattu miten sattuu eikä se valita mistään. Sekin pitäisi korjata jotta VSCode näyttäisi nämä suoraan punaisena.

Reply:
Aa joo, mä sekoilin ja poistin vain nuo mouseLeave handlerit. Nyt korvattu pilkut puolipisteellä